ank/doc/swdesign/README.md

```diff
@@ -529,6 +529,223 @@ Needs:
+| **tags** | It specifies a list use defined key/value objects. | A list of { `key`: some_string, `value`: some_string} or empty| true |
+| **dependencies** | It specifies inter workload dependencies. | Not specified yet and shall be set to empty.| false |
+| **updateStrategy** | It specifies the update strategy. | One of: `UNSPECIFIED`, `AT_LEAST_ONCE`, `AT_MOST_ONCE` | true |
+| **restart** | It specifies whether the workload shall be restarted when it exits | One of: `true`,`false` | true |
+| **accessRights** | It specifies lists of access rules fpr `allow` and `deny`. | Not fully specified yet and shall be set to empty list for both. | true |
+| **runtime** | It specifies the type of the runtime. | One of: `podman`, `podman-kube`. | true |
+| **runtimeConfig** | It specifies the configuration for the runtime whose configuration structure is specific for each runtime as a _string_ | As a _string_ from one of: [PodmanRuntimeConfig](#podmanruntimeconfig), [PodmanKubeRuntimeConfig](#podmankuberuntimeconfig),  | true |
+
+###### PodmanRuntimeConfig
+
+The runtime configuration for the podman runtime is specified as follows:
+
+```YAML
+generalOptions: [<comma>, <separated>, <options>]
+image: <registry>/<image name>:<version>
+commandOptions: [<comma>, <separated>, <options>]
+commandArgs: [<comma>, <separated>, <arguments>]
+```
+
+###### PodmanKubeRuntimeConfig
+
+The runtime configuration for the podman-kube runtime is specified as follows:
+
+```YAML
+generalOptions: [<comma>, <separated>, <options>]
+play_options: [<comma>, <separated>, <options>]
+down_options: [<comma>, <separated>, <options>]
+manifest: <string containing the K8s manifest>
+```
+
+#### CLI supports Ankaios manifest
+`swdd~cli-supports-ankaios-manifest~1`
+
+Status: approved
+
+The Ankaios CLI shall support the Ankaios manifest file format.
+
+Tags:
+- AnkaiosManifest
+
+Needs:
+- impl
+- utest
+
+#### CLI provides a function to accept a list of Ankaios manifest files
+`swdd~cli-apply-accepts-list-of-ankaios-manifests~1`
+
+Status: approved
+
+**When** the user provides a list of Ankaios manifest files via the CLI command `ank apply [OPTIONS] manifest1.yaml manifest2.yaml ...`,
+
+**Then** the Ankaios CLI shall accept the content of all the given Ankaios manifest files.
+
+Needs:
+- impl
+- utest
+- stest
+
+#### CLI provides a function to accept an Ankaios manifest content from `stdin`
+`swdd~cli-apply-accepts-ankaios-manifest-content-from-stdin~1`
+
+Status: approved
+
+**When** the user provides the manifest content via the CLI command `ank apply [OPTIONS] -` through `stdin`,
+
+**Then** the Ankaios CLI shall accept the given manifest content from `stdin`.
+
+Needs:
+- impl
+- utest
+- stest
+
+#### CLI provides a function to generate a state object from Ankaios manifests
+`swdd~cli-apply-generates-state-object-from-ankaios-manifests~1`
+
+Status: approved
+
+**Where** the user does not provide the optional argument `-d`,
```

Comment:
This requirement is not understandable to me. The first sentence with `Where` sounds to me rather like a condition (when/if). The requirement does not mention that we talk about "ank apply". Other requirements here are written in the format `Where ... and ..., somebody shall do something ... .
I am not the expert on writing requirements I have to say. It would be better to check this with our architects.